pep-3156.txt

```diff
@@ -361,7 +373,8 @@ stopping and closing.  (However, a partially-conforming event loop is
 still better than nothing. :-)
 
 - Starting, stopping and closing: ``run_forever()``,
-  ``run_until_complete()``, ``stop()``, ``is_running()``, ``close()``.
+  ``run_until_complete()``, ``stop()``, ``is_running()``, ``close()``,
```

Comment:
I'm saying `is_running` and friends are racy even in a single-threaded context, because there's no guarantee that the loop will keep running after you call it. In normal event loop usage, you know the loop is running because it's always running; if you have to ask whether it's running or not then you're doing something weird (and this is highly correlated with not understanding how event loops are supposed to work, although not 100%). 

If this is already in and people are relying on it despite it not being official, then so be it, but if this was going through a normal PEP-first process I'd argue against introducing these methods.